src/context/viewer-context.tsx

```diff
@@ -21,6 +31,7 @@ export type ViewerConfigOptions = {
   canvasHeight?: string;
   ignoreCaptionLabels?: string[];
   informationPanel?: {
+    autoScroll?: AutoScrollSettings | boolean;
```

Comment:
Should we be more specific about what is auto scrolling? Something like `vttAutoScroll` just to keep things organization a little better?

or even...

```js
vtt?: {
  autoScroll?: {
    ...
  }
}
```

Reply:
That's funny, I originally called it `autoScrollAnnotation` and then  `vttAutoScroll` before settling on `autoscroll`. I'm happy to change it or nest it; I was just trying to stay as simple and plain-English as possible.